offices/_single.html

```diff
@@ -165,16 +167,16 @@ <h2 class="h4 u-hide-on-mobile">
                     {% endif %}
                 </div>
             </div><!-- END .expandable -->
-        {% endfor %}
+        {% if loop.last %}
```

Comment:
Same question for the closing tags, checking number of pages vs wrapping in the loop